tests/test_custom_checkers/test_pycodestyle_checker.py

```diff
@@ -149,6 +149,22 @@ def test_no_error_e228(self) -> None:
         with self.assertNoMessages():
             self.checker.process_module(mod)
 
+    def test_error_e265(self) -> None:
```

Comment:
It looks like when resolving a merge conflict, you accidentally moved these two tests for E265. These tests shouldn't be moved, as this adds unnecessary changes to this diff. To fix this, look at their original location in the master branch and move them back to their correct position.